src/WebViewShared.ts

```diff
@@ -42,16 +40,16 @@ const createOnShouldStartLoadWithRequest = (
     url: string,
     lockIdentifier: number,
   ) => void,
-  originWhitelist: ?$ReadOnlyArray<string>,
```

Comment:
TypeScript also has a read-only array: `ReadonlyArray<T>` ☺️

Reply:
Well we do have eslint to not modify received parameters, but that's nice to know

Reply:
It's nice to use because if users are using readonly variables to enforce mutability, TypeScript won't let them pass those variables to us since it thinks we are going to mutate them.

Basically, it's a way to document the interface 📝

Reply:
ok nice I'll make those changes before merging :)